spec/ttml1.xml

```diff
@@ -5987,9 +5987,10 @@ as a style property only to those element types indicated in the following table
 overlap in space, the area(s) generated by lexically subsequent elements must be
 rendered over area(s) generated by lexically prior elements, where lexical
 order is defined as the postorder traversal of a <emph>Document Instance</emph>.</p>
+<p>The <el>tt</el> element is considered to establish the root of the stacking context, as defined by 
+<bibref ref="xsl11"/>, &sect; 7.30.18.</p>
 <p>The semantics of the value <code>auto</code> are those defined by
```

Comment:
With the proposed change, this paragraph can now be removed, since the definition of auto is already implied by the derivation note.

Reply:
@skynavga @nigelmegitt argues that the derivation note is informative and therefore cannot replace the normative clause _the semantics of the value auto are those defined by [XSL 1.1], § 7.30.18_

Reply:
I do not agree, because elsewhere, we do not normatively define in prose every keyword, but instead, rely upon the semantics of the derivation.

Reply:
I'm arguing for minimal changes here. The reference to XSL was already present here, but it was not correct. Merely editing it so that the scope is correct is the change needed to address this issue. If you want to make another change to modify the way that the semantic derivation is stated then that's another separate issue.

If you want to raise that [other issue], then consider that this reference appears to be normative whereas the derivation statement is in a note. I would also request that any such change is made broadly in line with w3c/ttml2#470.

Reply:
Having said that, I can see @skynavga 's point, and actually removing the statement about the semantics of `auto` altogether would not be harmful - I would not object to doing that, or to keeping it in.

Reply:
@nigelmegitt How would the reader determine the semantics of `auto` if the clause _The semantics of the value auto are those..._ is removed?

Reply:
@palemieux the same way the reader determines the semantics of `<integer>`!

Reply:
@nigelmegitt Alright.

Reply:
@skynavga Please approve the PR.